rust/main/chains/hyperlane-sealevel/src/rpc/client/tests.rs

```diff
@@ -0,0 +1,14 @@
+use crate::SealevelRpcClient;
+
+//#[tokio::test]
+async fn _test_get_block() {
```

Comment:
just wanna confirm this test is intentionally kept?

Reply:
Yes, I thought to keep some test code even if we don't run it. Future engineer will be able to use it with another block/slot.